dapp/src/pages/DashboardPage/AcreTVLProgress.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import {
+  StackProps,
+  useMultiStyleConfig,
+  HStack,
+  VStack,
+  Grid,
+  Box,
+} from "@chakra-ui/react"
+import { useTVL } from "#/hooks"
+import { BitcoinIcon } from "#/assets/icons"
+import { CurrencyBalance } from "#/components/shared/CurrencyBalance"
+import ProgressBar from "#/components/shared/ProgressBar"
+import { TextMd, TextXs } from "#/components/shared/Typography"
+
+type AcreTVLProgressProps = StackProps
+
+export function AcreTVLProgress(props: AcreTVLProgressProps) {
+  const styles = useMultiStyleConfig("AcreTVLProgress")
+  const totalValueLocked = useTVL()
+
+  return (
+    <Box sx={styles.container} {...props}>
+      <HStack sx={styles.wrapper}>
+        <Grid sx={styles.contentWrapper}>
+          <BitcoinIcon sx={styles.valueIcon} />
+
+          <CurrencyBalance
+            size="3xl"
+            amount={totalValueLocked.value}
+            currency="bitcoin"
+          />
+
+          <TextMd>Total value locked</TextMd>
+        </Grid>
+
+        <VStack sx={styles.progressWrapper}>
+          <HStack sx={styles.progressLabelsWrapper}>
+            {[50, 100, 150, 200, 250].map((value) => (
```

Comment:
I think these values should depend on `TOTAL_VALUE_LOCKED_CAP`. A quick solution that comes to my mind:

```
export const useTVL = () => {

  ...
  const totalValue = bigIntToUserAmount(
    TOTAL_VALUE_LOCKED_CAP,
    BITCOIN.decimals,
  )

  return { progress, value, isCapExceeded, totalValue }
}

```

```
const UNIT = 50

const { totalValue } = totalValueLocked
const scale = Array.from({ length: totalValue / UNIT },(_, idx) => totalValue - idx * UNIT).reverse()
```

Let me know what you think and if you may have any other solution.

Reply:
I discussed it with @nkuba.
The conclusion was to hardcode values.
Context [here](https://discord.com/channels/1079490158639976609/1297889075050582069/1300486504128446628)

Reply:
Let's assume we set the cap to 1250, so we have a step every 250. When we hit it and update the `TOTAL_VALUE_LOCKED_CAP` to 1600, the new progress bar will have a step every 320. So, the previous steps and the goal of 1250 are no longer displayed. This may be confusing as we don't maintain the historic goals.

I preferred hardcoding the steps, as depending on the change in the cap, we need to consider the steps to keep them consistent across updates.

We can keep the current automated calculation, but we need to be aware that we need to revisit the steps in case of an update in the future. It is worth adding a comment in the code.

Reply:
Let's assume we set the cap to 1250, so we have a step every 250. When we hit it and update the `TOTAL_VALUE_LOCKED_CAP` to 1600, the new progress bar will have a step every 320. So, the previous steps and the goal of 1250 are no longer displayed. This may be confusing as we don't maintain the historic goals.

I preferred hardcoding the steps, as depending on the change in the cap, we need to consider the steps to keep them consistent across updates.

We can keep the current automated calculation, but we need to be aware that we need to revisit the steps in case of an update in the future. It is worth adding a comment in the code.

Reply:
The number of steps on the progress bar is currently parametrized with a variable.

```ts
const STEP_COUNT = 5
```
- 4 steps: <img width="1098" alt="image" src="https://github.com/user-attachments/assets/f1fc5941-63cc-4629-b860-ddec00ff8d3c">

- 5 steps: <img width="1098" alt="image" src="https://github.com/user-attachments/assets/1f4d1837-aa58-4f0b-a50b-35e10beed4aa">

- 6 steps: <img width="1098" alt="image" src="https://github.com/user-attachments/assets/a9765649-a7b9-487a-b151-65649bdd3efe">

In my opinion it's enough control for now.  We can adjust it later in case more control is needed (like irregular steps you mentioned).